src/quo2/components/tags/status_tags.cljs

```diff
@@ -56,8 +56,8 @@
   [size theme label]
   [base-tag
    {:size             size
-    :background-color colors/success-50-opa-10
     :icon             :verified
+    :background-color colors/success-50-opa-10
     :border-color     colors/success-50-opa-20
     :label            label
     :text-color       (if (= theme :light)
```

Comment:
I think you can remove the parameter of `theme` from all of these functions and instead use `theme-colors`, e.g 
```
 { :text-color      (colors/theme-colors             
                      colors/success-50
                        colors/success-60)}
```

Reply:
There is an `override-theme` property that can be provided for `status-tag` externally and that's why there are all these `if (= theme :light)` conditions. There was an option to figure it out and fix but I intentionally decided to skip it because it's out of scope of this issue anyway.